azure-gradle-plugins-common/src/main/java/com/microsoft/azure/gradle/util/GradleAzureMessager.java

```diff
@@ -8,41 +8,37 @@
 import com.microsoft.azure.toolkit.lib.common.messager.IAzureMessage;
 import com.microsoft.azure.toolkit.lib.common.messager.IAzureMessager;
 import com.microsoft.azure.toolkit.lib.common.utils.TextUtils;
-import lombok.extern.slf4j.Slf4j;
+import lombok.AllArgsConstructor;
+import org.gradle.api.logging.Logger;
 
-@Slf4j
-public class GradleAzureMessager implements IAzureMessager {
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+@AllArgsConstructor
+public class GradleAzureMessager implements IAzureMessager, IAzureMessage.ValueDecorator {
+    private Logger log;
 
     @Override
     public boolean show(IAzureMessage message) {
         switch (message.getType()) {
             case ALERT:
             case CONFIRM:
             case WARNING:
-                log.warn(message.getMessage());
+                log.warn(message.getContent());
                 return true;
             case ERROR:
-                log.error(message.getMessage(), ((Throwable) message.getPayload()));
+                log.error(message.getContent(), ((Throwable) message.getPayload()));
                 return true;
             case INFO:
             case SUCCESS:
             default:
-                prompt(message.getMessage());
+                log.lifecycle(message.getContent());
                 return true;
         }
     }
 
-    public static void prompt(String message) {
-        // legacy code for prompt, will be replaced by new method: Notifier.noticeUser later
-        if (log.isInfoEnabled()) {
-            log.info(message);
-        } else {
-            System.out.println(message);
-        }
-    }
-
     @Override
-    public String value(String val) {
-        return TextUtils.cyan(val);
+    public String decorateValue(@Nonnull Object p, @Nullable IAzureMessage message) {
+        return TextUtils.cyan(p.toString());
```

Comment:
Shall we add a default impl for `decorateValue`? It seems that only IDE have different behavior

Reply:
assign miller to impl it in interface 'IAzureMessage.ValueDecorator', @wangmingliang-ms